extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1,56 +1,60 @@
 <?php
 
-/**
- * @package MediaWiki
- * @subpackage SpecialPage
- * @author Krzysztof Krzyżaniak <eloy@wikia-inc.com> for Wikia.com
- * @version: 0.1
- */
-
 if ( !defined( 'MEDIAWIKI' ) ) {
 	echo "This is MediaWiki extension and cannot be used standalone.\n";
 	exit( 1 ) ;
 }
 
-
 /**
  * @addtogroup SpecialPage
  */
 class WikiFactoryPage extends SpecialPage {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "WikiFactoryPage" has 22 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)